addons/gut/gui/gut_config_gui.gd

```diff
@@ -354,6 +354,8 @@ func set_options(options):
 	_add_title('Misc')
 	_add_value('prefix', options.prefix, 'Script Prefix',
 		"The filename prefix for all test scripts.")
+	_add_value('suffix', options.suffix, 'Script Suffix',
+		"The filename suffix for all test scripts.")
```

Comment:
Add "must end in .gd" to the description.

Reply:
Done.